test_schemas/en/test_section_summary.json

```diff
@@ -132,7 +132,10 @@
         {
             "id": "house-details-section",
             "title": "House Details Section",
-            "summary": { "show_on_completion": true },
+            "summary": {
+                "show_on_completion": true,
+                "title": "Household Summary"
```

Comment:
Given the requirement is to be able to have placeholders in this title, it might be worth adding placeholders to the test schema.

Reply:
D'oh